053-SentinelAutomatedResponse/Coach/Solution-03.md

```diff
@@ -0,0 +1,50 @@
+# Challenge-03 - Automated Response - Coach's Guide
+
+[< Previous Challenge](./Solution-02.md) - **[Home](./README.md)** 
+
+
+## Introduction
+In this section of the Coach's guide, you fill find guidance, code, and examples that will help you guide the teams over the course of the WTH. 
+In the spirit of continuous improvement, update this repository with any suggestions, altertnatives, or additional challenges.
```

Comment:
As per previous comments, include link to WTH Contribution guide here.  https://aka.ms/wthcontribute